pandapower/toolbox.py

```diff
@@ -2983,15 +2984,15 @@ def get_connected_buses(net, buses, consider=("l", "s", "t", "t3", "i"), respect
         else:
             opened_buses_hv = opened_buses_mv = opened_buses_lv = set()
 
-        hvb_trafos3w = set(net.trafo3w.index[
```

Comment:
Would not have been needed if Groups were already merged, but is fine to do it seperately :) (https://github.com/SteffenMeinecke/pandapower/blob/feature/groups/pandapower/toolbox.py#L2996-L3001)